go.mod

```diff
@@ -5,8 +5,8 @@ go 1.19
 replace github.com/google/go-jsonnet v0.19.1 => github.com/keboola/go-jsonnet v0.19.1
 
 require (
-	ariga.io/atlas v0.9.1-0.20230119145809-92243f7c55cb
-	entgo.io/ent v0.11.7
+	ariga.io/atlas v0.9.1
+	entgo.io/ent v0.11.9
```

Comment:
Ako mame uz komitnuty generovany kod, tak ak sa meni kniznica, co ten kod generuje, tak sa musi po update kod pre-generovat, aby a zistilo ci to ide/ci su nejake zmeny.

Asi ti nenapadlo zavolat `make generate-model`? alebo ano, to je tak kniznica, pre ten novy model.